tests/visco_plastic_adiabatic_pressure.prm

```diff
@@ -0,0 +1,107 @@
+# A test for using adiabatic pressure in creep viscosity calculation for
+# the visco_plastic material model.
```

Comment:
I find it really helpful to have some comments at the top of the test that provide a prediction of the expected values. For example, here you could take the adiabatic pressure at the top/bottom of the model and calculate (include equation in comment) what the expected viscosity values should be in the test.